apps/emqx_plugins/src/emqx_plugins.erl

```diff
@@ -253,6 +259,25 @@ write_package(NameVsn, Bin) ->
 is_package_present(NameVsn) ->
     emqx_plugins_fs:is_tar_present(NameVsn).
 
+-spec purge_other_versions(name_vsn()) -> ok.
+purge_other_versions(NameVsn) ->
```

Comment:
Maybe we should 
* call `list_plugins` here
* find plugin(s) with the same AppName
* call ensure_stopped, ensure_uninstalled

because the plugin that is being removed may actually be running